### PR TITLE
APEX-120 #comment using different check point location

### PR DESCRIPTION
**Pull request by gauravgopi123**

### CODE REVIEW

**Comment by gauravgopi123** [comment]
@vrozov Please review it

File: common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java [hunk 1]
```diff
@@ -63,7 +63,6 @@ protected void finished(Description description)
     {
       try {
         FileUtils.deleteDirectory(new File("target/" + description.getClassName()));
-        FileUtils.deleteDirectory(new File(FileUtils.getTempDirectory(), "localcheckpoint"));
```
Comment (vrozov) [inline common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java]: Can the created directory be deleted? Should directory be created as part of test setup and deleted in tear down?

File: common/src/main/java/com/datatorrent/common/util/AsyncFSStorageAgent.java [hunk 2]
```diff
@@ -49,9 +49,10 @@ public AsyncFSStorageAgent(String path, Configuration conf)
   {
     super(path, conf);
     try {
-      File tempFile = File.createTempFile("msp", "msp");
-      this.localBasePath = new File(tempFile.getParent(), "localcheckpoint").getAbsolutePath();
+      File tempFile = File.createTempFile("chkp", null);
       tempFile.delete();
+      tempFile.mkdir();
+      this.localBasePath = tempFile.getAbsolutePath();
```
Comment (vrozov) [inline common/src/main/java/com/datatorrent/common/util/AsyncFSStorageAgent.java]: Files.createTempDirectory?

File: common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java (hunk 1 above)
Comment (gauravgopi123) [inline common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java]: It is internally managed by storage agent so we can't delete it

Comment (vrozov) [inline common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java]: Will it be deleted when storage agent closes? Why it was possible to delete the directory before?

**Pull request merged**